packages/fx-core/src/plugins/resource/aad/permissions/permissions.json

```diff
@@ -53,1464 +53,1453 @@
           "allowedMemberTypes": [
             "Application"
           ],
-          "description": "Allow the app to evaluate the inputs provided against the Data Loss Prevention policies and recommend applicable actions, without a signed-in user.",
-          "displayName": "Evaluate Data Loss Prevention policy",
-          "id": "ba7b8302-40ad-475c-a768-5b990aa1dba1",
+          "description": "Allows\u00a0the\u00a0app\u00a0to\u00a0read\u00a0subject\u00a0rights requests\u00a0without a\u00a0signed-in\u00a0user.",
```

Comment:
is this change as desired?

Reply:
Yes, it is come from graph api response, and I have created an issue here:
https://github.com/microsoftgraph/microsoft-graph-docs/issues/17943